DependencyInjection/Configuration.php

```diff
@@ -83,6 +83,8 @@ public function getConfigTreeBuilder()
                             ->defaultValue('auto')
                         ->end()
                         ->scalarNode('formatter')->defaultNull()->end()
+                        ->booleanNode('inject_debugger_for_discovered_http_client')->defaultTrue()->end()
```

Comment:
Okay, so by default a discovered, unconfigured client is simply wrapped with debug tools and put back to discovery and you can also choose to use a configured client for that, but that is optional, opt-out by default?